azure-pipelines.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Microsoft Corporation.
+name: $(Date:yyyyMMdd)$(Rev:.r).0-$(SourceBranchName)
+
+pr:
+  - main
+
+trigger:
+  branches:
+    include:
+      - main
+
+jobs:
+  - job: BuildPublish
+    displayName: Build and publish
+    pool:
+      name: VSEngSS-MicroBuild2019-1ES
```

Comment:
Nit: I think for our purposes the 2022 pool should be fine, but it doesn't matter much either way.

Reply:
Changing.